src/fluent-api/tests/with-browser/with-browser.test.ts

```diff
@@ -44,6 +45,26 @@ describe('Playwright Fluent - withBrowser', (): void => {
     browserInstance && (await browserInstance.close());
   });
 
+  test('should target msedge', async (): Promise<void> => {
+    // Given
+    const browser = 'msedge';
+    const p = new PlaywrightFluent();
+
+    // When
+    await p.withBrowser(browser);
+
+    // Then
+    const browserInstance = p.currentBrowser();
+    const pageInstance = p.currentPage();
+    expect(browserInstance).toBeDefined();
+    expect(pageInstance).toBeDefined();
+
+    const userAgent =
+      pageInstance && (await pageInstance.evaluate(() => window.navigator.userAgent));
+    expect(userAgent).toContain('Edg');

```

Comment:
I just wondered why you test `Edg` and not `Edge`?

Reply:
Because userAgent looks next: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/96.0.4664.55 Safari/537.36 Edg/96.0.1054.43

Reply:
lol, it looks like to be either a bug or a way to bypass Microsoft licences :)

In this case I suggest to harden the condition to be `.toContain(' Edg/')`. 
In this way the test will pinpoint any modification of the user agent string that could be done in our back :)